Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM nvidia/cuda:11.3.0-cudnn8-runtime-ubuntu18.04
+
+RUN \
+    apt-get update && \
+    apt-get -y install wget \
+    aria2 \
+    rsync \
+    cmake \
+    build-essential \
+    curl \
+    git \
+    vim \
+    nvidia-cuda-toolkit
+
+# install miniconda
+RUN \
```

Comment:
General note -- why duplicate logic instead of just calling `install_third_party_dependencies.sh` which seems like the global install script.  If the issue is downloading of data flags could be added to that script to not download static assets.